js/src/pages/price-benchmark/price-benchmark-suggestions.js

```diff
@@ -122,6 +124,14 @@ const PriceBenchmarkSuggestions = () => {
 	const { suggestions, hasFinishedResolution } =
 		usePriceBenchmarkSuggestions();
 
+	if ( suggestions.length === 0 ) {
```

Comment:
@ankitrox We should also check if `hasFinishedResolution` is `true` to make sure that there's no data returned from the resolver.

Reply:
Added the check.